visualization/scatter_plot.js

```diff
@@ -60,14 +60,20 @@ const updateXAxis = () => {
 	svg.select('.x-axis').call(xAxis);
 };
 
-const updateScatterPlotYValues = (checkedAttributes) => {
-	normalizeSelectedAttributes(shownPlayers, checkedAttributes);
+const updateScatterPlotYValues = (sliderAttributes) => {
+	normalizeSelectedAttributes(shownPlayers, sliderAttributes);
 	yValue = function(d) {
 		// Calculate the combined score of each of the selected statistics
 		let combinedScore = 0;
-		for (let i = 0; i < checkedAttributes.length; i++) {
-			combinedScore += parseFloat(d[checkedAttributes[i] + '-Normalized']);
+		let norm_value = 0;
+		let weight_total = 0
+		for (var attr in sliderAttributes) {
+			norm_value = parseFloat(d[attr + '-Normalized'])
+			weight_value = parseFloat(sliderAttributes[attr]) / 100
+			weight_total += weight_value
+			combinedScore += (norm_value * weight_value);
 		}
+		combinedScore = combinedScore * (5/weight_total)

```

Comment:
I'm a bit confused on the purpose of: `* (5/weight_total)`, should this be the length of `sliderAttributes > 1` instead?

Reply:
Yeah the 5 only works because length of sliderAttributes is 5, should def just be # of sliderAttributes, I'll update that.  And yeah the idea was to normalize the max value to the # of attributes, so changing the weights changes their contribution, but not the max possible value.  Not sure if that's useful but that's what I was thinking, I'll leave it in for now.